### PR TITLE
i18n(es): Update of `astro-components.mdx`, `endpoints.mdx`, `layouts.mdx` & `client-side-scripts.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.

Update of `astro-components.mdx`, `endpoints.mdx`, `layouts.mdx` & `client-side-scripts.mdx` based on this commit -> 60f35d65a6ef99bb13fafbbc312c332213ab5a70

`Note:` `framework-components.mdx` has been updated already

### CODE REVIEW

File: src/content/docs/es/guides/client-side-scripts.mdx [hunk 1]
```diff
@@ -6,6 +6,27 @@ i18nReady: true
...
 
+## Scripts del lado del cliente
+
+Los scripts pueden ser usados para agregar event listeners, enviar datos de analítica, reproducir animaciones y todo lo que JavaScript puede hacer en la web.
+
+```astro
+// src/components/ConfettiButton.astro
+<button data-confetti-button>Celebrate!</button>
+<script>
+  // Importa modulos npm.
```
Comment (agustinmulet) [inline src/content/docs/es/guides/client-side-scripts.mdx:17]:
```suggestion
  // Importa módulos npm.
```

**Review by agustinmulet — CHANGES_REQUESTED** [review]
Un pequeño acento nada más. Después de eso, LGTM! 🚀

**Comment by dreyfus92** [comment]
Muchas gracias por el review @agustinmulet 😁

**Comment by Yan-Thomas** [comment]
Great job @dreyfus92 & @agustinmulet! Let's get this merged 🚀

**Pull request merged**